lib/src/main/java/uk/gov/di/ipv/core/library/helpers/RequestHelper.java

```diff
@@ -111,6 +111,14 @@ public static String getJourney(JourneyRequest request) {
         return request.getJourney();
     }
 
+    public static String getScoreType(ProcessRequest request) {
+        return request.getScoreType();
+    }
+
+    public static Integer getScoreThreshold(ProcessRequest request) {
+        return request.getScoreThreshold();
+    }
+
```

Comment:
I'm not sure there is much value in these helper methods - they're just proxies for the actual method we're interested in.